src/plone/restapi/services/navigation/get.py

```diff
@@ -29,7 +30,10 @@ def __init__(self, context, request):
 
     def __call__(self, expand=False):
         if self.request.form.get("expand.navigation.depth", False):
-            self.depth = int(self.request.form["expand.navigation.depth"])
+            try:
+                self.depth = int(self.request.form["expand.navigation.depth"])
+            except ValueError as e:
+                raise BadRequest(e)
```

Comment:
Could you write a test for this case please? The BadRequest gets serialized by plone.rest and I can't remember if you're allowed to pass another exception instance instead of a string.

Reply:
done